src/advanced_inv.cpp

```diff
@@ -471,8 +488,10 @@ int advanced_inv_area::get_item_count() const
 
 void advanced_inv_area::init()
 {
+    offz = 0; // FIXME: merely a placeholder until Z levels
```

Comment:
Mainly so it is noticeable being out of place (and with a comment beside it). I mean, it'll need to be updated after Coolthulu finishes zlevs, may as well make it stick out. :^)